ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -178,25 +180,26 @@ Item {
                 delegate: ChatContentView {
                     width: parent.width
                     clip: true
-                    height: {
-                        // dynamically calculate the height of the view, if the active one is the current one
-                        // then set the height to parent otherwise set it to 0
-                        if(!chatContentModule)
-                            return 0
-
-                        let myChatId = chatContentModule.getMyChatId()
-                        if(myChatId === root.activeChatId || myChatId === root.activeSubItemId)
-                            return parent.height
-
-                        return 0
-                    }
+                    height: isActiveChannel ? parent.height : 0
                     rootStore: root.rootStore
                     contactsStore: root.contactsStore
                     sendTransactionNoEnsModal: cmpSendTransactionNoEns
                     receiveTransactionModal: cmpReceiveTransaction
                     sendTransactionWithEnsModal: cmpSendTransactionWithEns
                     stickersLoaded: root.stickersLoaded
                     isBlocked: model.blocked
+                    isActiveChannel: {
```

Comment:
nit: can we create a function? Just realised I duplicated this code :facepalm: when writing it